zetaclient/bitcoin/bitcoin_client_live_test.go

```diff
@@ -348,3 +348,76 @@ func LiveTestAvgFeeRateTestnetMempoolSpace(t *testing.T) {
 
 	compareAvgFeeRate(t, client, startBlock, endBlock, true)
 }
+
+// Remove prefix "Live" to run this live test
```

Comment:
Can we rather use a build time rather than this?
Like hsm test
https://github.com/zeta-chain/node/blob/1e0d8593f02f5995237298c8a1ec79c58ad98f4a/zetaclient/hsm/hsm_signer_test.go#L1
https://github.com/zeta-chain/node/blob/1e0d8593f02f5995237298c8a1ec79c58ad98f4a/Makefile#L52

Reply:
That's a good option. Does the CI compile the code on different flags so that the code remains compilable after each PR?